src/Item.js

```diff
@@ -2,10 +2,48 @@ import React, { useState, useEffect } from 'react';
 import { db } from './lib/firebase';
 import estimates from './lib/estimates';
 import { differenceInDays, fromUnixTime } from 'date-fns';
+import { Modal } from '@material-ui/core';
+import styled from 'styled-components';
+
+const StyledModal = styled.div`
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 70%;
+  background-color: white;
+  padding: 6rem;
+  border-radius: 5px;
+  box-shadow: 0 3rem 5rem rgba(0, 0, 0, 0.3);
+  z-index: 10;
+  max-width: 250px;
+  font-size: 15px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const ModalBtn = styled.a`
+  display: inline-block;
+  padding: 5px;
+  margin: 0.5rem 1rem;
+  width: 6rem;
+  background: transparent;
+  color: black;
+  text-align: center;
+  cursor: pointer;
+  background: lightgray;
+  justify-content: center;
+  border: none;
+  &:hover {
+    background: darkgray;
+  }
+`;
 
 function Item({ userToken, item }) {
```

Comment:
At this point, most of the code in this file is for the modal and its functionality (deleting items from the database). Would it make sense to extract the modal into its own component?

Reply:
I agree that this is worth considering!

Reply:
This is true, I would think we could do that. If Karan agrees, we could go ahead and move it to a separate component.

Reply:
For sure! That's a great point!